directus-studio/index.md

```diff
@@ -0,0 +1,156 @@
+---
+title: "Building a Customer Support Ticket Form with Directus Studio"
+description: "Learn how to build a customer support ticket form with Directus Studio, enhancing service efficiency and user experience in an easy steps."
+author:
+  name: "Ayodele Aransiola"
+  avatar_file_name: "ayodele.png"
+---
+
+## Introduction
+
+Building a user-friendly and useable product is important, but an effective customer feedback and support is pivotal for business success. [Directus](https://directus.io), an open-source flexible platform that offers solutions such as the Headless CMS, Backend-as-a-service more amazing features that can be tailored to various needs, including creating a dynamic support ticket form for your product. This article will guide you through using Directus Studio to build a support ticket form that allows customers to submit their queries or issues on your product offerings.
+
+## Before You Start
+
+**:cloud: Instance**
+Imagine a scenario where your business receives numerous customer inquiries daily. Managing these inquiries efficiently becomes importantant. In this article, you will learn how [Directus](https://directus.io) can be helpful in building a seamless support ticket handling system for your business.
+<!-- ## Your Sections Here -->
+
+
+- Ensure that you have setup a Directus account either on the [Directus Cloud](https://directus.cloud/) or on your local development environment using [docker](https://docs.directus.io/getting-started/quickstart.html). Watch this youtube [video](https://www.youtube.com/watch?v=ZOCfBBTMtoE) for a step-by-step guide in setting up Directus locally on your computer.
+
+- Log into your Directus account, you'll create a specific collection for your support ticket, agent, and message. This collections will include the form for `Customer` to drop a ticket, the support collection will be for the `Support Agent` assigned to each ticket, while the message collection is for communication on the ticket. Below is the structure of the schema for each collections:
+
+	1. Support Ticket: Ticket_Id, Name, Email, Issue Type (options: Billing, Technical Issue, General Inquiry), Description, status (Open, In Progress, Resolved), asigned_to, and Submission Date.
+	2. Agents: agent_id (uuid), assigned (relationship: manyToOne, related collection: directus user)
+   3. Ticket Message: message_id, date_created, ticket_message, relationship (relationship: ManyToOne, related collection: support_tickets `ticket_id`)
+
+- Configuring roles for the support agents and the customers:
+	-  `Customers` can create tickets and view the status of their own tickets, as well as add message to the ticket.
+
+	-  `Support Agents` has access to view assigned tickets, update statuses, add message for the customer.
+
+Setting up these roles and permissions ensures that customers have a straightforward way to submit and track their inquiries while giving support agents the tools needed to manage and resolve tickets efficiently.
```

Comment:
It seems you break this down step by step below. As a result, I would delete the lightweight overview here. Perhaps replace it with a super high-level overview that just explains that there are three concepts: support tickets, agents assigned to tickets, and ticket messages.